first workflow ever

jooooscha opened the pull request.

jedahan commented:
the println! actually puts it in a nice spot

![2020-12-25_15-12-1608929442](https://user-images.githubusercontent.com/32407/103141881-03bcae80-46c9-11eb-9cd3-c288a6e3c3ba.jpg)

jooooscha commented:
I have implemented a proper message bar in 12ca8e0cc028dfec9b5d0f85793e532e04606832. It is called if an error occures.
Also, I added an option to disable the notify-send call (1def568dc8fabcd1cc5cef6c693f1e9948d0d05e)

I hope this also fixes your problems.

jedahan commented:
Wow looks fantastic, thank you!

jooooscha committed:
first workflow ever
